__tests__/integration/hathorwallet_facade.test.ts

```diff
@@ -1981,7 +1981,8 @@ describe('mintTokens', () => {
     const { hash: tokenUid } = await createTokenHelper(hWallet, 'Token to Mint', 'TMINT', 100);
 
     // Should not mint more tokens than the HTR funds allow
-    await expect(hWallet.mintTokens(tokenUid, 9000)).rejects.toThrow('HTR tokens');
+    await expect(hWallet.mintTokens(tokenUid, 9000))
+      .rejects.toThrow(/^Not enough HTR tokens for deposit: 90 required, \d+ available$/);
```

Comment:
The available tokens can change for each test, therefore we match for any digit with `\d+`.